document/core/appendix/implementation.rst

```diff
@@ -46,6 +46,8 @@ An implementation may impose restrictions on the following dimensions of a modul
 * the number of results in a :ref:`function type <syntax-functype>`
 * the number of parameters in a :ref:`block type <syntax-blocktype>`
 * the number of results in a :ref:`block type <syntax-blocktype>`
+* the number of parameters in a :ref:`tag type <syntax-tagtype>`
+* the total size of parameters in a :ref:`tag type <syntax-tagtype>`
```

Comment:
The "size" of parameters is not a well-defined notion for the spec, since e.g. reference types have no observable size. However, we can omit this item, since it is already subsumed by the number of parameters (for which the limit may vary with their types) – the same is already assumed for function parameters, locals, etc.

Reply:
ok. i removed the item.